tcms/testruns/templates/testruns/mutable.html

```diff
@@ -95,6 +95,59 @@
                 {{ form.build.errors }}
             </div>
         </div>
+        <div class="form-group">
+            <label class="col-md-1 col-lg-1" for="id_start_date">{% trans "Start date" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <div class="input-group date-time-picker-pf">
+                    <input type="text" class="form-control" id="id_start_date">
+                    <span class="input-group-addon">
+                        <span class="fa fa-calendar"></span>
+                    </span>
+                </div>
+
+                {% include "include/datetimepicker_script.html" with selector="#id_start_date" %}
+            </div>
+
+            <label class="col-md-1 col-lg-1" for="id_stop_date">{% trans "Stop date" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <div class="input-group date-time-picker-pf">
+                    <input type="text" class="form-control" id="id_stop_date">
+                    <span class="input-group-addon">
+                        <span class="fa fa-calendar">
+                        </span>
+                    </span>
+                </div>
+
+                {% include "include/datetimepicker_script.html" with selector="#id_stop_date" %}
+            </div>
+        </div>
+
+        <div class="form-group">
```

Comment:
These fields are tied to form fields on the backend for input validation. See definitions of other fields on this page and NewRunForm where you had made changes previously.